LICENSE.md

```diff
@@ -12,6 +12,22 @@ Mapbox Maps for Android version 10.0 ('Mapbox Maps Android SDK') or higher must
 For the full license terms, please see the Mapbox Terms of Service at https://www.mapbox.com/legal/tos/
 
 ## Additional Mapbox Maps Android licenses
+Mapbox Maps Android uses portions of the Activity.
+
+URL: [https://developer.android.com/jetpack/androidx/releases/activity#1.2.3](https://developer.android.com/jetpack/androidx/releases/activity#1.2.3)
+
+License: [The Apache Software License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+===========================================================================
+
+Mapbox Maps Android uses portions of the Activity.
+
+URL: [https://developer.android.com/jetpack/androidx/releases/savedstate#1.1.0](https://developer.android.com/jetpack/androidx/releases/savedstate#1.1.0)
```

Comment:
do we actually use savedstate dependency in some of our dependencies, is it just introduced because of the version bump?

Reply:
I think these are introduced because we bumped the license version. Our sdk doesn't use these, it could be that some dependencies uses this dependency.

Reply:
these are now changed after downgrade the plugin version to `0.8.80`